src/pybind/mgr/balancer/module.py

```diff
@@ -348,11 +352,32 @@ def show_status(self) -> Tuple[int, str, str]:
         }
         return (0, json.dumps(s, indent=4, sort_keys=True), '')
 
+    @CLIReadCommand('balancer status detailed')
```

Comment:
```suggestion
    @CLIReadCommand('balancer status detail')
```

Let's call this command "ceph balancer status detail" since that is more in line with other ceph detail commands.